src/classes/ADDR_CopyAddrHHObjBTN_TEST.cls

```diff
@@ -200,6 +200,55 @@ private with sharing class ADDR_CopyAddrHHObjBTN_TEST {
         System.assert(mockPerms.updateSObjFields.contains(npo02__Household__c.npo02__MailingStreet__c));
     }
 
+    /***
+    * @description Verifies an error message is displayed when a user does not have household update permissions
+    * @return void
+    */
+    @isTest
+    private static void shouldErrorWhenBtnClickedWithoutCreateAddressPerms() {
```

Comment:
- The ApexDoc is not in sync with this test
- Suggestion so other tests naming is followed: `shouldErrorWhenBtnClickedWithoutAddressCreatePerm`
- Let's remove the `addAddress` on line 209 and use `List<Address__c> defaultAddresses = getAddressRecord();`.